src/import/crds-dev.ts

```diff
@@ -17,26 +14,12 @@ import { ImportCustomResourceDefinition, ManifestObjectDefinition } from './crd'
 /**
  * Matches a https://doc.crds.dev repo
  *
- *  - ManifestObjectDefinition[] if found
+ *  - url if found
  *  - undefined if not
  *
  * @param source
  */
-export async function importCrdsDevRepoMatch(importSpec: ImportSpec): Promise<undefined | ManifestObjectDefinition[]> {
```

Comment:
A consequence of changing the `ImportCustomResourceDefinition` constructor to private.